requirements.txt

```diff
@@ -1,2 +1,2 @@
-setuptools==41.0.1
+setuptools==42.0.2
```

Comment:
I am a bit hesitant about this setuptools version. But it should be okay. It will give warnings, but that can be a good thing.
See this [buildout issue](https://github.com/buildout/buildout/issues/493#issuecomment-558334008)